src/autoload/global.gd

```diff
@@ -147,8 +147,7 @@ func load_config() -> void:
 	if "window_size" in config:
 		var raw_size := config["window_size"] as Vector2i
 		var fixed_size: Vector2i
-		@warning_ignore("integer_division")
-		if raw_size.x > 3 * raw_size.y / 4:
+		if 4 * raw_size.x > 3 * raw_size.y:
```

Comment:
![image](https://github.com/kdreese/gmtk-2023/assets/22134557/2534aba0-7cae-4257-a768-0f782e8b12af)

Reply:
me when the
![image](https://github.com/kdreese/gmtk-2023/assets/7307599/78b722b7-6853-4d73-a9c1-16c26db22292)